src/components/pagination/pagination.jsx

```diff
@@ -148,18 +143,16 @@ PaginationNext.displayName = 'Pagination.Next';
 const PaginationEllipsis = ( props ) => {
 	const { size, disabled } = usePageContext();
 	return (
-		<li
-			tabIndex={ 0 }
-			className={ cn( 'flex', disabled && disabledClassNames.general ) }
-		>
+		<li className={ cn( 'flex', disabled && disabledClassNames.general ) }>
 			<span
 				className={ cn(
+					'flex justify-center',
 					sizeClassNames[ size ].ellipse,
 					disabled && disabledClassNames.general
 				) }
 				{ ...props }
 			>
-				...
+				•••
```

Comment:
Please use a lucide Icon.

Reply:
Akbar suggested that I use three dots like this.